### DISCUSSION TITLE
Migration to springboot

### DISCUSSION THREAD

**Discussion by kavinkumarc** [dropwizard/dropwizard#4785] [General]
Can anyone refer any websites or help me in converting the dropwizard code to springboot?
I require spring boot code for the following libraries:
1)  io.dropwizard.Application;
2) io.dropwizard.jetty.MutableServletContextHandler;
3) io.dropwizard.setup.Bootstrap;
4) io.dropwizard.setup.Environment;

**Comment by joschi** [comment]
@kavinkumarc You might be better asking this in one of the Spring community support channels. 😉 

➡️ https://spring.io/community
